jenkins-scripts/docker/lib/_ros_setup_buildsh.bash

```diff
@@ -123,6 +124,12 @@ for d in \$DIRS; do
 done
 echo '# END SECTION'
 
+# clean up build and install directories to save disk space in Jenkins server
+if [[ ! $KEEP_WS_FILES ]]; then
+  rm -fr ${CATKIN_WS}/install/*
+  rm -fr ${CATKIN_WS}/build/*
```

Comment:
Do we need to remove `devel` when using catkin?

Reply:
`colcon` makes a `log` folder too

Reply:
This  is subt-pr_any- job in drogon:
```
   1.4 GiB [##########] /install
  139.9 MiB [          ] /build
   88.2 MiB [          ] /devel
    4.0 KiB [          ] /src
    4.0 KiB [          ]  .catkin_workspace
```
and ros2-ci-pr_any-foxy
```                         /..                                                                                                                                                    
  516.0 MiB [##########] /build
    6.4 MiB [          ] /install
    5.1 MiB [          ] /log
    8.0 KiB [          ] /src
```
There is at least one order of magnitude of difference between install/build and the rest. No problem on including more things.

Reply:
that's fair; I've approved as is